bridge/hardhat/script/changeOwner.js

```diff
@@ -0,0 +1,130 @@
+// How to run the script: npx hardhat run ./hardhat/script/createRifToken.js --network bsctestnet
+const hre = require("hardhat");
+
+async function main() {
+    const {getNamedAccounts, deployments} = hre;
+    const {deployer} = await getNamedAccounts();
+    const network = hre.network.name;
+
+    let newWalletAddress = null;
+
+    switch (network){
+        case "rsktestnet":
+            newWalletAddress = '0x51591ec8f296ef3e4deebe32c392c706aef42133';
```

Comment:
these address applicable for any network? i mean they are  constant?

Reply:
They are not, there is a switch there is one per network.